timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -15,6 +15,8 @@
 
 namespace timemachine {
 
+static const int NOISE_PER_STEP = 2;
```

Comment:
document why this is 2?

Reply:
Documented in https://github.com/proteneer/timemachine/pull/1225/commits/8caa3f77dfc2dcc87b2411cc4dde881b4111d464